backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -273,4 +279,42 @@ public static Extension convertToTribalAffiliationExtension(String tribalAffilia
     }
     return null;
   }
+
+  public static ServiceRequest convertToServiceRequest(TestOrder order) {
+    if (order != null) {
+      ServiceRequestStatus serviceRequestStatus = null;
+      switch (order.getOrderStatus()) {
+        case PENDING:
+          serviceRequestStatus = ServiceRequestStatus.ACTIVE;
+          break;
+        case COMPLETED:
+          serviceRequestStatus = ServiceRequestStatus.COMPLETED;
+          break;
+        case CANCELED:
```

Comment:
Just a note that I don't think this case will ever be hit - if a test order is cancelled we don't try to report it to ReportStream. Definitely still worth having this case though! I just don't anticipate we'll use it in practice.